.junie/skills/tailwindcss-development/SKILL.md

```
@@ -0,0 +1,129 @@
+    <div>Card 2</div>
+    <div>Card 3</div>
+</div>
+```
+
+## Common Pitfalls
+
+- Using deprecated v3 utilities (bg-opacity-*, flex-shrink-*, etc.)
+- Using `@tailwind` directives instead of `@import "tailwindcss"`
+- Trying to use `tailwind.config.js` instead of CSS `@theme` directive
+- Using margins for spacing between siblings instead of gap utilities
+- Forgetting to add dark mode variants when the project uses dark mode
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)